src/Discovery/DiscoveryMDSSearch.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Input,
+} from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+
+class DiscoveryMDSSearch extends React.Component {
+  render() {
```

Comment:
Since this only has a render() should it be a functional component? Or do we generally avoid them for some reason?